EIPS/eip-4675.md

```diff
@@ -0,0 +1,212 @@
+---
+eip: 4675
+title: Multi-Fractional Non-Fungible Token Standard
+description: Token standard to handle multiple NFTs in a fractionalized form
+author: David Kim (@powerstream3604)
+discussions-to: https://ethereum-magicians.org/t/eip-4675-multi-fractional-non-fungible-token-standard/8008
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-13
+requires: 165, 721
+---
+
+## Abstract
+This standard outlines a smart contract interface eligible to represent any number of fractionalized non-fungible tokens. Existing projects utilizing standards like EIP-1633 conventionally deploy seperate ERC-20 compatible token standard to fractionalize the non-fungible token into ERC-20 tokens. In contrast, this ERC allows each token ID to represent a token type representing(fractionalizing) the non-fungible token.
```

Comment:
The first mention of another EIP must be linked, use EIP instead of ERC, and I think you had the wrong word in here:

```suggestion
This standard outlines a smart contract interface eligible to represent any number of fractionalized non-fungible tokens. Existing projects utilizing standards like [EIP-1633](./eip-1633.md) conventionally deploy separate [EIP-20](./eip-20.md) compatible token contracts to fractionalize the non-fungible token into EIP-20 tokens. In contrast, this ERC allows each token ID to represent a token type representing(fractionalizing) the non-fungible token.
```